topics:
  sub:
    raw_pc: "/livox/lidar_192_168_10_114/pc"
  pub:
    segmented_pc: "/livox/lidar_192_168_10_114/segmented_pc"

frames:
  source: "shovel"
  reference: "world"

distance_error_threshold: 0.2
path_to_cad: "/home/aravindh/Semester Project/data/SHOVEL.stl"
